src/common/library.h

```diff
@@ -3,7 +3,16 @@
 #ifndef FF_INCLUDED_common_library
 #define FF_INCLUDED_common_library
 
-#include <dlfcn.h>
+#if defined(_WIN32) //We don't force MSYS to use LoadLibrary because dlopen also searches $LD_LIBRARY_PATH
+    #include <libloaderapi.h>
+    // Use #define to avoid `RTLD_LAZY is not defined` errors
+    #define dlopen(path, mode) ((void*)LoadLibraryA((path)))
+    #define dlsym(handle, symbol) ((void*)GetProcAddress((HMODULE)(handle), (symbol)))
```

Comment:
I would prefer to have them as static inline methods, we can just define `RTDL_LAZY` to anything and FF_UNUSED it in the method.